vm/src/obj/objint.rs

```diff
@@ -188,7 +189,7 @@ impl PyInt {
 
     #[pymethod(name = "__sizeof__")]
     fn sizeof(&self, _vm: &VirtualMachine) -> usize {
-        self.value.bits() + 7 & !7
+        mem::size_of::<Self>() + (self.value.bits() + 7) & !7
```

Comment:
```python
40
>>>>> (2**8).__sizeof__()
48
>>>>> (2**16).__sizeof__()
56
>>>>> (2**32).__sizeof__()
72
```

Reply:
`__sizeof__` should actually return bytes, so you'd probably want to divide the bits you get by 8